constants/index.js

```diff
@@ -6,7 +6,6 @@ module.exports = {
     datetime: 'DD MMMM YYYY HH:mm'
   },
   ropsYears: [
-    2022,
```

Comment:
I think the entire PR can be replaced by some logic here that adds an extra year.

For future-proofness I'd also automatically have it include all years from 2021 until the current year, with the flag to add "next" year.

Reply:
cool - done